### ISSUE TITLE
Feature Request: Cache next/previous image files in CBZ / comic format

### ISSUE THREAD

**Issue by breaker27** [sumatrapdfreader/sumatrapdf#680]
I have many scanned documents saved as cbz files.

It would be nice if SumatraPDF would read the next and previous 1-10 (configurable) amount of images in the background. So if I scroll with the mouse through the pages, it won't flicker anymore, but show the images promptly. Maybe instead of the max. number of cached pages, a max. cache size in MB would be also helpful.

This could be implemented with a caching thread that permanently reads next/previous pages from the current one.

**Comment by GitHubRulesOK** [comment]
@kjk 
as I understand it SumatraPDF already pre-loads Mb of multiple pages already, however my experience of stutter was when internal file types kept changing through the comic e.g mixed filetypes png jpg j2k etc.

**Comment by kjk** [comment]
Yes, we do pre-render next / prev page. That being said, I'm sure caching of images could be improved, especially in .cbz etc. files, so let's keep this open.

**Comment by GitHubRulesOK** [comment]
@kjk ping but this could be open for ever as you must have optimised hundreds of times 💯 since 2020 and over the past 20 years :-)

**Comment by kjk** [comment]
Yeah, I would need a file and a scenario that needs to be improved. Otherwise it's too vague.
